tests/cli/test_cli_add_family.py

```diff
@@ -2,10 +2,11 @@
 from datetime import datetime
 
 import click
-from cg.constants import Pipeline
+from cg.constants import Pipeline, DataDelivery
 from cg.store import Store
 
 CLI_OPTION_ANALYSIS = Pipeline.BALSAMIC
+CLI_OPTION_DELIVERY = DataDelivery.QC
 
 
 def test_add_family_required(invoke_cli, disk_store: Store):
```

Comment:
No chance of switching to case here and the corresponding sub that is tested?

Reply:
That would change the CLI and I think that's beyond the scope of this PR. Let's do change it later tough